master/internal/stream/system.go

```diff
@@ -295,6 +306,8 @@ func doPublishLoop[T stream.Msg](
 		// The pq listener example includes a timeout case, so we do too.
 		// (https://pkg.go.dev/github.com/lib/pq/example/listen)
 		case <-time.After(30 * time.Second):
+			// TODO: look into handling return value of Ping()
```

Comment:
can you do me a favor?

While we are working on this feature, our codebase is free of `XXX` comments, so when we want to mark something as "this is wrong, I know, let's come back to it", use `XXX` and not `TODO`?

Then at the end we can `git grep -I XXX` and find all the comments we still need to address.

In contrast, there are a small nonzero number of reasons why somebody would land a normal `TODO` comment in the code, so they don't stand out nearly as much.